### ISSUE TITLE
Terminal minimum contrast ratio not met for dom renderer

### ISSUE THREAD

**Issue by Sehun-Kiim** [Issue-Labeling-Bot/Labeling_Bot#98]
Issue Type: Bug

Testing #85966

Use WSL on windows
Set terminal.integrated.minimumContrastRatio to 21
Run true color test from https://unix.stackexchange.com/questions/404414/print-true-color-24-bit-test-pattern/404415#404415
Using the canvas renderer, white and black text are used at this contrast ratio:
true_color_canvas

But using the webgl and dom renderer, the true color test is still low contrast:
true_color_webgl

VS Code version: Code - Insiders 1.41.0-insider (9785578, 2019-12-03T05:39:36.001Z)
OS version: Windows_NT ia32 10.0.18363

**Comment by Choi-kwang-hyun** [comment]
bug : 0.4225146472454071

**Comment by Choi-kwang-hyun** [comment]
verified : 0.40734341740608215
